pkg/operator/ceph/disruption/register_controllers.go

```diff
@@ -16,25 +16,36 @@ limitations under the License.
 
 // Package controllers contains all the controller-runtime controllers and
 // exports
-package controllers
+package disruption
 
 import (
 	"fmt"
 
-	"github.com/rook/rook/pkg/operator/ceph/disruption/controllerconfig"
-
 	"github.com/rook/rook/pkg/operator/ceph/disruption/clusterdisruption"
+	"github.com/rook/rook/pkg/operator/ceph/disruption/controllerconfig"
+	"github.com/rook/rook/pkg/operator/ceph/disruption/machinedisruption"
+	"github.com/rook/rook/pkg/operator/ceph/disruption/machinelabel"
 	"github.com/rook/rook/pkg/operator/ceph/disruption/nodedrain"
 
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 )
 
+var (
+	EnableFencing bool
```

Comment:
`FencingManagementEnabled` ?

Reply:
In the code I think `EnableFencing` reads more smoothly, but I'm fine either way.